tests/integrations/test_init.py

```diff
@@ -0,0 +1,26 @@
+from sentry_sdk.integrations import Integration, setup_integrations
+
+
+class NoOpIntegration(Integration):
+    """
+    A simple no-op integration for testing purposes.
+    """
+
+    identifier = "noop"
+
+    def setup_once():  # type: () -> None
+        pass
+
+    def __eq__(self, __value: object) -> bool:
+        """
+        All instances of NoOpIntegration should be considered equal to each other.
+        """
+        return type(__value) == type(self)
+
+
+def test_multiple_setup_integrations_calls():
```

Comment:
@sentrivana Do you think this new file is an appropriate place for this test?

Reply:
Why not just put it in `tests/test_basics.py` since we already have some integration tests in there?

Or maybe we could rename this new file from `test_init.py` to something more generic like `test_common.py` and also move other integrations-related tests from `test_basics.py` here.

Reply:
Okay, I'll put it in `test_basics.py` for now. Maybe in the future we could split out into a separate file, but I think moving everything over would be out of scope for this issue